src/Comparer/StubValueComparer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Diff\Comparer;
+
+/**
+ * @since 3.0
+ *
+ * @license GPL-2.0+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class StubValueComparer implements ValueComparer {
```

Comment:
Do you have an alternative proposal?

I moved it out of the production code folder for now, though think this is not in the best interest of the users of this library. Realistically speaking it doubt it will make much difference either way so no point spending much time debating it.